src/Lykke.Job.BlockchainCashoutProcessor/Wrokflow/CommandHandlers/StartCashoutCommandsHandler.cs

```diff
@@ -49,19 +53,40 @@ public class StartCashoutCommandsHandler
                 throw new InvalidOperationException("BlockchainIntegrationLayerAssetId of the asset is not configured");
             }
 
-            var hotWaletAddress = _hotWalletProvider.GetHotWalletAddress(asset.BlockchainIntegrationLayerId);
+            string toAddress = command.ToAddress;
+            string hotWaletAddress = _hotWalletProvider.GetHotWalletAddress(asset.BlockchainIntegrationLayerId);
+            Guid? clientId = await _walletsClient.TryGetClientIdAsync(asset.BlockchainIntegrationLayerId,
```

Comment:
Name it recipientClient, targetClient or something like this, to make it intention more clear

Reply:
fixed